tests/python/test_errors.py

```diff
@@ -32,90 +32,90 @@
 from smartredis.error import *
 
 
-def test_SSDB_not_set(use_cluster, context):
+def test_SSDB_not_set(context):
     ssdb = os.environ["SSDB"]
     del os.environ["SSDB"]
     with pytest.raises(RedisConnectionError):
-        c = Client(None, use_cluster, logger_name=context)
+        c = Client(None, logger_name=context)
     os.environ["SSDB"] = ssdb
 
 
-def test_bad_SSDB(use_cluster, context):
+def test_bad_SSDB(context):
     ssdb = os.environ["SSDB"]
     del os.environ["SSDB"]
     os.environ["SSDB"] = "not-an-address:6379;"
     with pytest.raises(RedisConnectionError):
-        c = Client(None, use_cluster, logger_name=context)
+        c = Client(None, logger_name=context)
     os.environ["SSDB"] = ssdb
 
 
-def test_bad_get_tensor(use_cluster, context):
-    c = Client(None, use_cluster, logger_name=context)
+def test_bad_get_tensor(context):
+    c = Client(None, logger_name=context)
     with pytest.raises(RedisReplyError):
         c.get_tensor("not-a-key")
 
 
-def test_bad_get_dataset(use_cluster, context):
-    c = Client(None, use_cluster, logger_name=context)
+def test_bad_get_dataset(context):
+    c = Client(None, logger_name=context)
```

Comment:
Do we have any tests that still use the old `Client(cluser: bool, address: str, logger_name: str)` style constructor? If not, should we keep/add one until we formally remove that constructor?

Reply:
There is one in address.py that does!